[MP-1726] Update Message Stream examples repo (iOS)

angeloesquibel opened the pull request:
This PR:
- Rebrands every reference of carnival into marigold

https://sailthru.atlassian.net/browse/MP-1726

angeloesquibel committed:
[MP-1726] Update Message Stream examples repo (iOS)

This PR:
- Rebrands every reference of carnival into marigold

https://sailthru.atlassian.net/browse/MP-1726

igstewart3 commented on iOS/MarigoldStreamExamples/MarigoldStreamExamples/AppDelegate.m:
```diff
@@ -18,7 +18,7 @@ @implementation AppDelegate
     
     // Override point for customization after application launch.
-    [Carnival startEngine:@"f0f9e7185392a99a09403d9dc000ed35b1758794"];
+    [[Marigold new] startEngine:@"6c566d865f4d647ab6f2c9f2653d670090751b80" error:nil];
```
Since this is a public repo, we should avoid including any SDK keys here. Maybe change it to @"<YOUR-SDK-KEY>" or similar.

igstewart3 commented on iOS/MarigoldStreamExamples/MarigoldStreamExamples/ListStreamViewController.m:
```diff
@@ -158,11 +158,11 @@ - (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)
-        //Register the message as deleted with Carnival
-        //[CarnivalMessageStream removeMessage:message withResponse:NULL]; //TODO: Uncomment this when you're happy with how the stream deletion looks and works.
+        //Register the message as deleted with Marigold
+        //[MarigoldMessageStream removeMessage:message withResponse:NULL]; //TODO: Uncomment this when you're happy with how the stream deletion looks and works.
```
Typo - this should be MARMessageStream

igstewart3 commented on iOS/MarigoldSwiftStreamExamples/MarigoldSwiftStreamExamples/AppDelegate.swift:
```diff
@@ -1,21 +1,26 @@
-        Carnival.startEngine("f0f9e7185392a99a09403d9dc000ed35b1758794")
+        // Override point for customization after application launch.        
+        do {
+            try Marigold().startEngine("6c566d865f4d647ab6f2c9f2653d670090751b80")
```
Same SDK key issue here, please replace.

angeloesquibel committed:
[MP-1726] Update Message Stream examples repo (iOS)

This Commit:

Remove references of sdk keys and fix typo error

https://sailthru.atlassian.net/browse/MP-1726

igstewart3 commented on iOS/MarigoldStreamExamples.xcworkspace/xcuserdata/affian.xcuserdatad/UserInterfaceState.xcuserstate:
We should remove all `xcuserdata` from the repo

angeloesquibel committed:
[MP-1726] Update Message Stream examples repo (iOS)

This PR:

- Rebrands every reference of carnival into marigold

https://sailthru.atlassian.net/browse/MP-1726

igstewart3 reviewed (changes requested):
Looks like there's still some xcuserdata committed, and some .DS_Store files as well. Can we strip these out to avoid unnecessary files in the repo?

angeloesquibel committed:
[MP-1726] Update Message Stream examples repo (iOS)

This PR:

Rebrands every reference of carnival into marigold
https://sailthru.atlassian.net/browse/MP-1726

angeloesquibel committed:
[MP-1726] Update Message Stream examples repo (iOS)

This PR:

Rebrands every reference of carnival into marigold
https://sailthru.atlassian.net/browse/MP-1726

angeloesquibel commented:
Should I remove the .DS_Store file from the base folder as well?
